Controllers/HomeController.cs

```diff
@@ -3,21 +3,90 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using System.Text.RegularExpressions;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
 
 namespace Zoltu.Bags.Client.Controllers
 {
 	public class HomeController : Controller
 	{
+		// Best practices
+		// https://www.asp.net/web-api/overview/advanced/calling-a-web-api-from-a-net-client
```

Comment:
Style: Thanks for the comment with the link, very helpful!  Can you put it all on one line though?
` // Best Practices: https://www.asp.net/web-api/overview/advanced/calling-a-web-api-from-a-net-client`